service/http.proxy/src/test/scripts/org/kaazing/gateway/service/http/proxy/auth/http.proxy.auth.forbidden.status.code.rpt

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# curl 7.37.1 <---> Gateway <---> tomcat 8.0.1
+
+property invalidCredentials ${http:loginBase64Encoder("admin:admin")}
+property authHeader ${http:append("Basic ", invalidCredentials)}
+connect http://localhost:8110/resource
+connected
+
+write method "GET"
+write header "Authorization" ${authHeader}
+write close
+
+read status "403" "Forbidden"
+
+
+#
+# tomcat server
+#
+
+property invalidCredentials ${http:loginBase64Encoder("admin:admin")}
```

Comment:
@vstratan No need to duplicate this properties. Only one per .rpt file is enough.